UI/SplashScreenSample/SplashScreenSample.Mobile/iOS/LaunchScreen.storyboard

```diff
@@ -1,52 +1,44 @@
-﻿<?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="11762" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" colorMatched="YES" initialViewController="5" launchScreen="YES">
+<?xml version="1.0" encoding="UTF-8"?>
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="21507" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" launchScreen="YES" colorMatched="YES" initialViewController="5">
```

Comment:
Did we need to modify the whole storyboard - aren't we just changing the image path?

Reply:
I opened on xcode, so it updated the rest, I presume... I can revert it and edit the `xml` file if you want